rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +99,24 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+* If a `Set` RPC contains `origin` CLI, the CLI origin MUST be given precedence
+  over the OpenConfig origin.  This precedence is independent of the order in
```

Comment:
Over any other origin? Remember that there are origins that can be defined by the vendor, or other projects. I believe there are origins used for elements of P4 information (or there were intended to be previously).

Reply:
I would like to contain the scope of this clarification to setRequest containing a mix of OpenConfig and CLI origins.   

In a future PR, we could consider additional combinations.  Please feel free to reopen this thread if you think this will cause a conflict or leads to some invalidation of the OpenConfig + CLI behavior defined here.